Assets/Scripts/Gameplay/Player/PlayerController.cs

```diff
@@ -7,10 +7,10 @@ public class PlayerController : MonoBehaviour
         [SerializeField] private CharacterController _characterController;
         private Vector3 _motion = Vector3.zero;
         public InventoryObject inventory;
-        //public ItemPickup itemPickup;
         [SerializeField] private GameObject _lantern;
-        [SerializeField] private GameObject _lanternProp;
         [SerializeField] private GameObject _exitRocks;
+        [SerializeField] private UI _ui;
+        private bool _equipped;
```

Comment:
Bool naming convention is not respected.
Plus, `_equipped` is not self-explanatory.